ursa.ino

```diff
@@ -402,7 +402,7 @@ void zeroMPU6050() {  // find how much offset each gyro axis has to zero out dri
       digitalWrite(LED_BUILTIN, i % 2);
       delay(25);
     }
-  } while (abs(rotationOffsetX) > 100 || abs(rotationOffsetY) > 100 || abs(rotationOffsetZ) > 100);
+  } while (abs(rotationOffsetX) > 500 || abs(rotationOffsetY) > 500 || abs(rotationOffsetZ) > 500);
```

Comment:
might be good to make this a variable

Reply:
I don't plan to change these numbers again, should they still be a variable?

Reply:
Probably, just for readability's sake.

Reply:
what do you think of 
`while (max(max(abs(rotationOffsetX),abs(rotationOffsetY)),abs(rotationOffsetZ)) > 500);`
so 500 is only written once?

Reply:
on second thought, that is less readable, I'm pushing a commit using variables